backend/src/main/java/com/bakdata/conquery/resources/admin/rest/AdminProcessor.java

```diff
@@ -189,8 +188,17 @@ public void addImport(Dataset dataset, File selectedFile) throws IOException {
 			TableId tableName = new TableId(dataset.getId(), header.getTable());
 			Table table = dataset.getTables().getOrFail(tableName);
 
+			final ImportId importId = new ImportId(table.getId(), header.getName());
```

Comment:
Kannst du hier bitte noch einen kleinen Kommentar ranmachen was `header.getName()` normalerweise ist?
Ich schätze es ist `tableName.tag` , aber dadurch, dass dieses Feld beim Preprocessing mal gesetzt wird ist es ziemlich weit weg. 
Oder ein Kommentar in `PreprocessedHeader`